scripts/pylib/twister/twisterlib.py

```diff
@@ -782,13 +782,22 @@ def run_custom_script(script, timeout):
 
     def handle(self):
         runner = None
+        timeout_cnt = os.getenv('TWISTER_DEVICE_TIMEOUT')
```

Comment:
we should not depend on environment variables, need some other approach to feed this data.